src/app/store/kubernetes-url-generator.service.ts

```diff
@@ -43,7 +43,7 @@ export class KubernetesUrlGenerator {
     }
   }
 
-  getEntityList(entityName: string, op: Operation, queryParams?: QueryParams | string): string {
+  getEntityList(entityName: string, op: HttpMethods, queryParams?: QueryParams | string): string {
```

Comment:
As far as I can see, the `op` parameter is not used at all, so you could just remove it

Reply:
Right, my IDE didn't show this :(